corehq/apps/app_manager/views/apps.py

```diff
@@ -490,11 +490,17 @@ def replace_all(list1, list2):
 
 @require_can_edit_apps
 @no_conflict_require_POST
-def edit_app_translations(request, domain, app_id):
+def edit_app_ui_translations(request, domain, app_id):
     params = json_request(request.POST)
     lang = params.get('lang')
     translations = params.get('translations')
     app = get_app(domain, app_id)
+
+    # Workaround for https://github.com/dimagi/commcare-hq/pull/10951#issuecomment-203978552
+    # auto-fill UI translations might have modules.m0 in the update originating from popular-translations docs
+    # since module.m0 is not a UI string, don't update modules.m0 in UI translations
+    translations.pop('modules.m0', None)
```

Comment:
It fetches translations from [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/translations/models.py#L78) which fetches translations from the view called `app_translations_by_popularity/view` (which presumably returns most popular translation for a key)